### PR TITLE
Adding a blog on Falco performance testing with event-generator

### PR DESCRIPTION
**Pull request by SamarSidharth**
Adding a blog for Falco performance testing using event-generator

Signed-off-by: SamarSidharth <samar.sidharth89@gmail.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

/kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area blog

> /area documentation

> /area videos

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: content/en/blog/Falco-Performance-Testing.md [hunk 1]
```diff
@@ -0,0 +1,327 @@
+|           Title           |                         Description                          |    Date    | Author         |
+| :-----------------------: | :----------------------------------------------------------: | :--------: | -------------- |
+| Falco Performance Testing | Understanding Falco performance testing with event-generator | 2021-01-20 | Samar Sidharth |
+
```
Comment (leogr) [inline content/en/blog/Falco-Performance-Testing.md:4]:
```suggestion
---
title: Falco Performance Testing
description: Understanding Falco performance testing with event-generator
date: 2020-09-23
author: 2021-01-20
slug: falco-performance-testing
---

```

File: content/en/blog/Falco-Performance-Testing.md [hunk 2]
```diff
@@ -0,0 +1,327 @@
+|           Title           |                         Description                          |    Date    | Author         |
+| :-----------------------: | :----------------------------------------------------------: | :--------: | -------------- |
+| Falco Performance Testing | Understanding Falco performance testing with event-generator | 2021-01-20 | Samar Sidharth |
+
+# Falco Performance Testing
+
+**Special Thanks to Leonardo Grasso from Sysdig for assisting me**
```
Comment (leogr) [inline content/en/blog/Falco-Performance-Testing.md:7]:
```suggestion
**Special Thanks to [Leonardo Grasso](https://github.com/leogr) for assisting me**

```
:heart:

File: content/en/blog/Falco-Performance-Testing.md [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+  INFO syscall.ChangeThreadNamespace                 actual=1613 expected=1613 ratio=1
+  INFO syscall.ReadSensitiveFileUntrusted            actual=1615 expected=1615 ratio=1
+  INFO statistics                                    cpu="22.4%" lost="0%" res_mem="121 MB" throughput="968.4 EPS" virt_mem="1.5 GB"
+  
+  ```
+
+
+
+
+
```
Comment (leogr) [inline content/en/blog/Falco-Performance-Testing.md:327]:
```suggestion
```

**Review by leogr — COMMENTED** [review]
First quick review, just found some minor issues.

Thank you so much for this very detailed blog :hugs:  

I will take a look more deeply soon!

File: content/en/blog/Falco-Performance-Testing.md (hunk 3 above)
Comment (SamarSidharth) [reply content/en/blog/Falco-Performance-Testing.md]: Thanks @leogr :)

**Review by fntlnz — APPROVED** [review]

**Comment by fntlnz** [comment]
Adding an hold to let @leogr look at this again and approve. Leo please remove it when you are done!

File: content/en/blog/falco-performance-testing.md [hunk 4]
```diff
@@ -0,0 +1,325 @@
...
+
+|    CPU    | Memory | Number of syscalls per second | EPS (rounded to lower limit) |
+| :-------: | :----: | :---------------------------: | :--------------------------: |
+| 500m(0.5) | 512 Mi |           Upto 150K           |             2800             |
+|     1     | 512 Mi |           Upto 250K           |             4200             |
+|     2     | 512 Mi |           Upto 320K           |             7000             |
+
+
+
+**Important  points:**
```
Comment (leodido) [inline content/en/blog/falco-performance-testing.md:291]:
```suggestion
**Important points:**

```

**Review by leodido — APPROVED** [review]
Great effort! Thank you two for this blog post :)

I've noticed some multiple white spaces and multiple whitelines around the markdown.
Suggested one to remove.

**Review by leogr — APPROVED** [review]
LGTM

PS: I'm aware of there're some whitespace issues that we can fix later

/hold cancel

**Pull request merged**